### DISCUSSION TITLE
Is there a way to disable the Console.KeyAvailable in the Console Loop? (Dedicated Server)

### DISCUSSION THREAD

**Discussion by Hecatohedron** [SmartlyDressedGames/Unturned-3.x-Community#3040] [Q&A] [answered]
I am writing a utility for managing Unturned servers and am having issues writing to standard input to run commands. After some digging I think this could be due to the `Console.KeyAvailable` in the dedicated server's code sent [here](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/957#issuecomment-517383895) by Nelson.

Is there a way to turn this functionality off or am I misunderstanding the issue entirely? 
I've tried using the `-LegacyConsole` argument but it didn't seem to have any effect.

Thank you for your time!

**Comment by educatalan02** [comment, marked as answer]
Take a look here: https://github.com/Raxul257/UnturnedConsoleModule
